plugin-server/src/utils/db/utils.ts

```diff
@@ -99,28 +106,20 @@ export function personInitialAndUTMProperties(properties: Properties): Propertie
     return propertiesCopy
 }
 
-export function generateKafkaPersonUpdateMessage(
-    createdAt: DateTime | string,
-    properties: Properties,
-    teamId: number,
-    isIdentified: boolean,
-    id: string,
-    version: number,
-    isDeleted = 0
-): ProducerRecord {
+export function generateKafkaPersonUpdateMessage(person: Person, isDeleted = false): ProducerRecord {
     return {
         topic: KAFKA_PERSON,
         messages: [
             {
                 value: JSON.stringify({
-                    id,
-                    created_at: castTimestampOrNow(createdAt, TimestampFormat.ClickHouseSecondPrecision),
-                    properties: JSON.stringify(properties),
-                    team_id: teamId,
-                    is_identified: isIdentified,
-                    is_deleted: isDeleted,
-                    ...(version !== null ? { version } : {}),
-                }),
+                    id: person.uuid,
+                    created_at: castTimestampOrNow(person.created_at, TimestampFormat.ClickHouseSecondPrecision),
+                    properties: JSON.stringify(person.properties),
+                    team_id: person.team_id,
+                    is_identified: Number(person.is_identified),
```

Comment:
note: This line actually *did* change, but it brings the type into alignment with the `ClickHousePerson` type and ClickHouse schema where it's defined as `Int8`: https://github.com/PostHog/posthog/blob/9f838f90b9a1c8fcb6d25a0f099449f06c87bdf3/posthog/models/person/sql.py#L30